packages/hash/frontend/src/theme/palette.ts

```diff
@@ -35,6 +35,8 @@ export const customColors = {
     80: "#5532C3",
     90: "#4625AA",
     100: "#3A2084",
+    gradient:
+      "linear-gradient(82.89deg, #9E56FA 5.64%, #644CFF 78.19%, #0070F4 121.05%)",
```

Comment:
we shouldn't add gradients here since it will be difficult to add all possible gradients in the app. The gradient should be placed in the sx property of the component it's being used.
If possible, we can do something like

```
<Box sx={{ background: ({ palette }) => `linear-gradient(82.89deg, ${palette.blue[50] 5.64%, ... })` }}></Box>
```

Reply:
This gradient applies to all primary button variants - [Figma link](https://www.figma.com/file/gydVGka9FjNEg9E2STwhi2/HASH-Editor?node-id=179%3A3548).

I'll inline it in the buttons theme file instead.